src/axios/apiQueries.js

```diff
@@ -42,3 +42,15 @@ export const postComment = (review_id = 3, username = "grumpy19", body) => {
       console.log(data);
     });
 };
+
+export const getListAllCategories = () => {
+  return api.get(`/categories`).then(({ data }) => {
+    return data.categories;
+  });
+};
+
+export const getReviewsSingleCategory = (category) => {
+  return api.get(`/reviews?category=${category}`).then(({ data }) => {
```

Comment:
This function is also doing what the getReviews function is doing. Axios has the ability to pull out params if there any. For example, if there is a category, it can make a request to the same /reviews endpoint, however, it will also append the category query to it as well so the backend can respond with all reviews from a chosen category